### PR TITLE
Replace Revision dropdowns with default select

### PR DESCRIPTION
**Pull request by chidozieononiwu**
Temporarily removing the custom dropdown select due to performance issues.

Should resolve https://github.com/Azure/azure-sdk-tools/issues/5151

### CODE REVIEW

**Review by konrad-jamrozik — APPROVED** [review]

File: src/dotnet/APIView/APIViewWeb/Pages/Assemblies/Review.cshtml [hunk 1]
```diff
@@ -384,18 +383,18 @@
                             {
                                 if (@Model.DiffRevisionId == @revision.RevisionId)
                                 {
-                                    <option style="font-size: 0.8em;" value="@urlValue" selected data-content="@optionName"></option>
+                                        <option value="@urlValue" selected>@optionName</option>
                                 }
                                 else
                                 {
-                                    <option style="font-size: 0.8em;" value="@urlValue" data-content="@optionName"></option>
+                                        <option value="@urlValue">@optionName</option>
```
Comment (weshaggard) [inline src/dotnet/APIView/APIViewWeb/Pages/Assemblies/Review.cshtml:390]: Is the extra level of indention intentional?

Comment (chidozieononiwu) [reply src/dotnet/APIView/APIViewWeb/Pages/Assemblies/Review.cshtml]: Nope, I have fixed it.

File: src/dotnet/APIView/APIViewWeb/Pages/Assemblies/Review.cshtml [hunk 2]
```diff
@@ -317,23 +317,23 @@
...
             <div class="form-group col-md-6 mb-0">
                 <label class="mb-0 ml-1"><small>Revision:</small></label>
-                <select class="selectpicker show-tick show-menu-arrow shadow-sm" data-style="btn-light btn-sm border" data-selected-text-format="value" data-live-search="true" data-size="10" data-width="100%" data-container="body" id="revisions-bootstraps-select" data-tick-icon="fa-solid fa-check">
+                <select class="custom-select shadow-sm custom-select-sm" id="revisions-bootstraps-select">
                     @foreach (var revision in Model.Review.Revisions.Reverse())
                     {
-                        var approvedBadge = "<i class='fas fa-check-circle text-success ml-2'></i>";
-                        var optionName = revision.IsApproved ? $"{@revision.DisplayName} {@approvedBadge}" : @revision.DisplayName;
-                        var urlValue = @Url.ActionLink("Review", "Assemblies", new {
+                        var optionName = revision.IsApproved ? $"{@revision.DisplayName} (APPROVED)" : @revision.DisplayName;
```
Comment (praveenkuttappan) [inline src/dotnet/APIView/APIViewWeb/Pages/Assemblies/Review.cshtml:323]: So it's going to show APPROVED label instead of green tick?

Comment (chidozieononiwu) [reply src/dotnet/APIView/APIViewWeb/Pages/Assemblies/Review.cshtml:323]: Yes for now.

**Pull request merged**